person.rb

```diff
@@ -3,6 +3,7 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rental'
 
 # here is the top level documentation
 class Person < Nameable
```

Comment:
- [ ] it would be better to add `rentals = [ ]`  so that it will be initialized instead of  adding from rental file

Reply:
Thanks for the review I have Implemented the required changes kindly review it again 👏  @youmari

Reply:
- [ ] Kindly add rentals as the previous TSE requested, you can do it that way:

`class Person < Nameable
  def initialize(age, name = 'Unknown', parent_permission: true)
    super()
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
   @rentals = [ ]
  end`